### PR TITLE
bug 1732829 can't scale empty MachineSets

### PR DESCRIPTION
**Pull request by kalexand-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1732829

@sunzhaohua2, will you PTAL?

### CODE REVIEW

File: modules/machineset-manually-scaling.adoc [hunk 1]
```diff
@@ -38,8 +43,7 @@ machines to be available.
 +
 [IMPORTANT]
 ====
-By default, the {product-title} router pods are deployed on workers.
-Because the router is required to access some cluster resources, including the
-web console, do not scale the worker MachineSet to `1` or `0` unless you first
-relocate the router pods.
+By default, the {product-title} router pods are deployed on workers. Because the router is required to access some cluster resources, including the web console, do not scale the worker MachineSet to `1` unless you first relocate the router pods.
+
```
Comment (jhou1) [inline modules/machineset-manually-scaling.adoc]: > do not scale the worker MachineSet to `1` unless you first relocate the router pods.

I guess you wanted to say do not scale it to `0`.

**Comment by jhou** [comment]
@kalexand-rh - Seems like the correct `jhou`  ( @jhou1 ) is on this :) 

may I ask that you remove me from the description so I don't get updates ?   Thank you!

**Comment by jhou1** [comment]
LGTM

File: modules/machineset-manually-scaling.adoc [hunk 2]
```diff
@@ -7,6 +7,11 @@
 
 If you must add or remove an instance of a machine in a MachineSet, you can manually scale the MachineSet.
 
+[IMPORTANT]
+====
+You cannot scale a MachineSet up if it currently contains `0` machines.
+====
+
```
Comment (jhou1) [inline modules/machineset-manually-scaling.adoc]: @kalexand-rh We can scale a machineset up from `0` **manually**. 
The issue with https://bugzilla.redhat.com/show_bug.cgi?id=1732829 is the autoscaler can not scale it up from `0`. So I think we can remove the statement `You cannot scale a MachineSet up if it currently contains `0` machines.` from the machineset-manually-scaling section and only highlight it in the applying-autoscaling section.

Sorry for the turn arounds.

File: modules/machineset-manually-scaling.adoc [hunk 3]
```diff
@@ -38,8 +43,7 @@ machines to be available.
 +
 [IMPORTANT]
 ====
-By default, the {product-title} router pods are deployed on workers.
-Because the router is required to access some cluster resources, including the
-web console, do not scale the worker MachineSet to `1` or `0` unless you first
-relocate the router pods.
+By default, the {product-title} router pods are deployed on workers. Because the router is required to access some cluster resources, including the web console, do not scale the worker MachineSet to `0` unless you first relocate the router pods.
+
+Because you cannot scale a MachineSet up from `0`, you must not scale down to `0`.
```
Comment (jhou1) [inline modules/machineset-manually-scaling.adoc]: Same as previous comment

**Comment by kalexand-rh** [comment]
@jhou1, will you PTAL?

**Comment by jhou1** [comment]
LGTM

**Comment by kalexand-rh** [comment]
Thank you! Merging.

**Pull request merged**